pkg/odo/cli/component/create.go

```diff
@@ -849,7 +855,7 @@ func (co *CreateOptions) Validate() (err error) {
 		return nil
 	}
 
-	log.Info("Validation")
+	log.Info("Validation for S2I component")
```

Comment:
Just keep it as `Validation`, I don't think we should declare that it's S2I.. As that would confuse the user.